src/transformers/models/llama/configuration_llama.py

```diff
@@ -125,6 +125,7 @@ def __init__(
         tie_word_embeddings=False,
         rope_theta=10000.0,
         rope_scaling=None,
+        bias=False,
```

Comment:
let's document this, and use `attention_bias` or something more expressive

Reply:
Sure!

Reply:
On the same note the PR could be rename to '[`InternLM`] Add support for InternLM' 😉

Reply:
Both done!